oci/artefact.go

```diff
@@ -341,11 +349,51 @@ func (c *Client) PushArtefact(ctx context.Context, destinationRef, sourceDir str
 		return "", fmt.Errorf("pushing index failed: %w", err)
 	}
 
-	if err := remote.Tag(tagAlias, index, c.remoteWithContext(ctx)...); err != nil {
-		return "", fmt.Errorf("adding alias tagging failed: %w", err)
+	for i := range tagAliases {
+		if err := remote.Tag(tagAliases[i], index, c.remoteWithContext(ctx)...); err != nil {
+			return "", fmt.Errorf("adding alias tagging failed: %w", err)
+		}
+	}
+	// TODO: reteurn tag and all of its aliases
```

Comment:
Coming back to this PR, this looks like the one outstading item. It only reutnrs first tag ingnoring all of the rest of tags.